src/oge/reference_tables/temporary_primary_fuel_manual.csv

```diff
@@ -1,3 +1,6 @@
 plant_id_eia,subplant_id,energy_source_code,subplant_primary_fuel,plant_primary_fuel,year,notes
-65373,7,NG,NG,NG,2023,CEMS unit ID but no EIA generator ID
-65373,8,NG,NG,NG,2023,CEMS unit ID but no EIA generator ID
\ No newline at end of file
+65373,7,NG,NG,NG,2023,Missing in EIA 860
+65373,8,NG,NG,NG,2023,Missing in EIA 860
+1554,2,RFO,RFO,RFO,2023,Fuel missing for this generator
```

Comment:
This unit was recently retired. Updating the primary fuel code to pull in capacity-based primary fuel for recently retired units.